Makefile

```diff
@@ -34,6 +34,7 @@ $(APP_NAME): $(TARGET) $(APP_TEMPLATE)
 	@mkdir -p $(APP_BINARY_DIR)
 	@cp -fRp $(APP_TEMPLATE) $(APP_DIR)
 	@cp -fp $(APP_BINARY) $(APP_BINARY_DIR)
+	@ln -s /Applications $(APP_DIR)
```

Comment:
Wouldn't this be part of the `install` target? I generally feel pretty cautious about touching a user's directories inside a `Makefile`.